packages/client/hmi-client/src/utils/number.ts

```diff
@@ -0,0 +1,159 @@
+/**
+ * Cleans a number string by removing non-numeric characters.
+ *
+ * @param {string} num - The number string to clean.
+ * @returns {string} The cleaned number string.
+ */
+export function cleanNumberString(num: string): string {
```

Comment:
Why not just use the native `parseFloat`, since what you are doing is pretty much, well, parsing a float. 

The native implementation has additional features to deal with scientific notations and other things, which yours does not.

Reply:
got rid of this function